### PR TITLE
Add support to scopes

### PR DESCRIPTION
**Pull request by venikman**
Part of https://github.com/noirdoor/noirdoor/issues/73

### CODE REVIEW

**Comment by venikman** [comment]
https://github.com/noirdoor/noirdoor/issues/73

File: index.js [hunk 1]
```diff
@@ -55,7 +55,15 @@ const register = (server, option) => {
             if (auth.isAuthenticated) {
                 // Credentials also have: .expiresIn, .token, .refreshToken
                 // Put the Auth0 profile in a cookie. The browser may ignore it If it is too big.
-                request.cookieAuth.set({ user : auth.credentials.profile });
+                if (auth.credentials.profile.raw.scope) {
```
Comment (sholladay) [inline index.js]: There is no need for the `if ()` statement here. The `scope` either exists or it doesn't, the `if` doesn't do anything.

**Comment by sholladay** [comment]
I still think it would be better to use `validateFunc`. Using this approach, the user will have the same scopes for the lifetime of their session, even if we change them on the backend. It would be better if the server checked that the user still has a scope in the database, for each request.

File: index.js (hunk 1 above)
Comment (sholladay) [inline index.js:58]: There is no need for the `if ()` statement here. The `scope` either exists or it doesn't, the `if` doesn't do anything.

**Review by sholladay — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (venikman) [reply index.js:58]: with if statement I have less errors on server. I checked it .

**Review by venikman — COMMENTED** [review]
Ok. If you want to use `validateFunc` I will do it. And closing that PR.

**Comment by venikman** [comment]
Ok.

**Pull request closed without merging**